feat: support token provided in an environment variable (#102)
jenkinsci/snyk-security-scanner-plugin: README.md

Snyk Installation Name. As configured in "[2. Configure a Snyk Installation](#2-configure-a-snyk-installation)".

#### `snykTokenId` (optional, default: *none*)

Snyk API Token Credential ID. As configured in "[3. Configure a Snyk API Token Credential](#3-configure-a-snyk-api-token-credential)".

> If you prefer to provide the Snyk API Token another way, such using alternative credential bindings, you'll need to
> provide a "SNYK_TOKEN" build environment variable.

Reviewer: I'd simplify this a bit and say something like:

```suggestion
Alternatively, set an environment variable `SNYK_TOKEN` containing the Snyk API token.
```

Author: Simplifying here may be confusing as credential bindings don't explicitly mention using environment variables. CI maintainers may not know that "environment variables" also means they can use credential bindings (see screenshot in PR).

Also environment variables in general are ambiguous (there are multiple layers), specifically here, the "build" prefix matters to ensure build agents are the ones receiving env vars and not, say, the master. It's not really "build [environment variables]" but rather "[build environment] variables".
